_episodes/04-function-style.md

```diff
@@ -543,28 +631,38 @@ $ git push origin master
 ~~~
 {: .bash}
 
-There are other tools, such as [pylint](https://www.pylint.org/) or [flake8](https://flake8.pycqa.org/en/latest/) which are not automatic formatters, but will check your code for adherence to the PEP8 standard. Pylint, for example, will find your variables which are not `snake_case`, functions which do not have `docstrings`, simple stylistic changes, unused variables, etc. Flake8 is a little less strict in general. We will try `flake8` out here.  If you would like to try `flake8`, first install it
+There are other tools, such as [pylint](https://www.pylint.org/) and
+[flake8](https://flake8.pycqa.org/en/latest/) that are not automatic formatters,
+but will check your code for adherence to the PEP8 standard.
+Pylint, for example, will find your variables which are not `snake_case`,
+functions which do not have `docstrings`, simple stylistic changes, unused variables, etc.
+Flake8 is a little less strict in general.
+We will try `flake8` out here.
+If you would like to try `flake8`, first install it.
 
 ~~~
 $ pip install flake8
 ~~~
 {: .language-bash}
 
-You can run it on our `functions` module:
+You can run it on our `functions` module.
 
 ~~~
 $ flake8 molecool/functions.py
 ~~~
 {: .language-bash}
 
-To see any errors still left in the module. Let's examine one of these 
+shows any errors still left in the module. Let's examine one of these.
```

Comment:
```suggestion
Let's examine one of the errors shown by the flake8 command above.
```

Reply:
@janash please feel free to apply this suggestion if it looks good to you, or mark "resolved" and offer an alternative.